apps/full-site-editing/full-site-editing-plugin/full-site-editing-plugin.php

```diff
@@ -127,9 +151,8 @@ function normalize_theme_slug( $theme_slug ) {
 }
```

Comment:
it looks like this function `normalize_theme_slug` was added twice, which causes an error and fails the branch loading locally.

Reply:
holy cow, good catch! fixed. It must have gotten in during a rebase. (speaking of, I have now rebased this branch :))